prime-router/metadata/fhir_mapping/hl7/datatypes/HD/Identifier/IdentifierUniversalId.yml

```diff
@@ -12,6 +12,6 @@ extension:
   expressionsMap:
     url:
       type: SYSTEM_URL
-      value: identifier-universal-id
-    valueBoolean:
-      valueOf: "true"
\ No newline at end of file
+      value: hl7v2Field
+    valueString:
+      valueOf: "HD.2,HD.3"
```

Comment:
I'd like some feedback on this pattern.  The alternative would be adding two extensions with each having it's own field.

Reply:
hmmmmmm, both options kinda suck lol. This patten is shorter but could trip up some FHIR senders one day. The other option adds a second unnecessary extension. So I vote for keeping this pattern.